File: chat_v2_message_counter_sample/src/main/java/com/zendesk/chat_v2/sample/MainActivity.java
package com.zendesk.chat_v2.sample;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import zendesk.chat.ChatConfiguration;
import zendesk.chat.ChatEngine;
import zendesk.messaging.MessagingActivity;

public class MainActivity extends AppCompatActivity implements UnreadMessageCounter.UnreadMessageCounterListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        if (Global.isMissingCredentials()) {
            setContentView(R.layout.missing_credentials);
            return;
        }
        setContentView(R.layout.activity_main);
        final UnreadMessageCounter unreadMessageCounter = new UnreadMessageCounter(this);

        findViewById(R.id.btnChat).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                unreadMessageCounter.stopCounter();
                MessagingActivity.builder()
                        .withEngines(ChatEngine.engine())
                        .show(MainActivity.this);

            }
        });

        findViewById(R.id.btnCounter).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                unreadMessageCounter.startCounter();
            }
        });
    }

    @Override
    public void onUnreadCountUpdated(int unreadCount) {
        ((TextView) findViewById(R.id.tvMessageCounter)).setText(Integer.toString(unreadCount));
    }
}

File: support_override_styles/src/main/java/bfahy/support_override_styles/MainActivity.kt
package bfahy.support_override_styles

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import zendesk.support.guide.HelpCenterActivity
import zendesk.support.requestlist.RequestListActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (Global.isMissingCredentials()) {
            setContentView(R.layout.missing_credentials)
            return
        }
        setContentView(R.layout.activity_main)

        helpCenterBtn.setOnClickListener {
            HelpCenterActivity
                    .builder()
                    .show(this@MainActivity)
        }

        requestListBtn.setOnClickListener {
            RequestListActivity
                    .builder()
                    .show(this@MainActivity)
        }
    }
}
